ios/PolyPodApp/PolyPod/AppDelegate.swift

```diff
@@ -23,9 +26,54 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let count = try! managedContext.count(for: fetchRequest)
         print("Number of quads in Core Data:", count)
         
+        self.registerUpdateNotificationCheck()
+        
         return true
     }
     
+    private func registerUpdateNotificationCheck() {
+        BGTaskScheduler.shared.register(forTaskWithIdentifier: AppDelegate.updateNotificationCheckIdentifier, using: nil) { task in
```

Comment:
linting nit: I guess `task in` should go to the next line.

Reply:
That looks super weird the way Xcode auto indents, basically:

```
foo() {
    bar in
    doSomething(bar)
}
```

So I usually leave it after the opening brace for better readability.